templates/vue/src/assets/styles/tyde-colors.scss

```diff
@@ -0,0 +1,3 @@
+$tyde-orange: #f79621;
+$tyde-orange-light: #f9b664;
+$tyde-blue: #1074bb;
```

Comment:
These don't seem to be used so this whole file should be removed. In addition, we shouldn't use color names (e.g. "blue") to name colors, but rather "primary" "secondary" etc so we can later change them to different colors if needed.